File: engine/engine-model/src/main/scala/net/server/channel/packet/maker/MakerDisassemblingPacket.scala
package net.server.channel.packet.maker

class MakerDisassemblingPacket(private var _type: Int, private var _toCreate: Int, private var _inventoryType: Int, private var _position: Int) extends BaseMakerActionPacket(_type = _type, _toCreate = _toCreate) {
  def inventoryType: Int = _inventoryType

  def position: Int = _position
}

File: engine/engine-model/src/main/java/net/server/channel/packet/maker/MakerDisassemblingPacket.java
package net.server.channel.packet.maker;

public class MakerDisassemblingPacket extends BaseMakerActionPacket {
   private final Integer inventoryType;

   private final Integer position;

   public MakerDisassemblingPacket(Integer theType, Integer toCreate, Integer inventoryType, Integer position) {
      super(theType, toCreate);
      this.inventoryType = inventoryType;
      this.position = position;
   }

   public Integer inventoryType() {
      return inventoryType;
   }

   public Integer position() {
      return position;
   }
}
